mslice/presenters/cut_plotter_presenter.py

```diff
@@ -97,8 +97,28 @@ def hide_overplot_line(self, workspace, key):
             line = cache.pop(key)
             remove_line(line)
 
-    def add_overplot_line(self, workspace_name, key, recoil, cif=None):
-        recoil = False
+    @staticmethod
+    def _get_log_bragg_y_coords(size, portion_of_axes, datum):
+        datum = 0.001 if datum == 0 else datum
+        y1, y2 = plt.gca().get_ylim()
+        if (y2 > 0 and y1 > 0) or (y2 < 0 and y1 < 0):
+            total_steps = np.log10(y2 / y1)
+        elif y1 < 0:
+            y1_int = -1
+            y2_int = 1
+            total_steps_up = np.log10(y2 / y2_int) + 1 if abs(y2) >= 1 else abs(y2)
+            total_steps_down = np.log10(y1 / y1_int) + 1 if abs(y1) >= 1 else abs(y1)
```

Comment:
```suggestion
            total_steps_up = np.log10(y2) + 1 if abs(y2) >= 1 else abs(y2)
            total_steps_down = np.log10(-y1) + 1 if abs(y1) >= 1 else abs(y1)
```

Reply:
Good point, though out of interest would you still follow what this bit of code is trying to do after the change?

Reply:
For me, I would probably find it easier to understand without the `_int` variables